youtube_dl/extractor/fczenit.py

```diff
@@ -0,0 +1,52 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import os.path
+import re
+import json
+
+from ..compat import compat_urllib_parse_unquote
+from ..utils import url_basename
+from .common import InfoExtractor
+
+class fczenitIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://(?:www\.)?fc-zenit\.ru/video/gl(?P<id>[0-9]+))'
+    _TEST = {
+    u'url': u'http://fc-zenit.ru/video/gl6785/',
```

Comment:
Since you used `from __future__ import unicode_literals`, you don't need the `u` prefix.